source/documentation/device_wifi.md

```diff
@@ -0,0 +1,28 @@
+# Certificate Authentication (Alpha)
+
+## What is Certificate Authentication
+Device-Wifi requires a user to have either a government email address, be sponsored by someone with a government email address, or have a mobile phone number. Device-Wifi only requires that a certificate needs to be installed on a device to gain access, this removes the need to always enter an email and password to gain access to government Wifi.
+
+## How it works
+1. Register your organisation with the GovWifi Admin portal and setup the “GovWifi” SSID on your local network.
+2. Setup a “Certificate Authority” using a Public Key Infrastructure (PKI) software tool. This allows the organisation to sign certificates.
+3. Using your PKI, sign client certificates.
+4. Provide us with your root CA certificate used to sign these client certificates.
+5. Devices can now connect to the internet via GovWifi wherever GovWifi is offered.
```

Comment:
Could add something about the user selecting EAP-TLS rather than EAP-PEAP, eg: 
![Screenshot 2019-06-05 at 14 17 31](https://user-images.githubusercontent.com/2160769/58959293-adadb080-879c-11e9-8d0e-24f1fe6cbe58.png)